src/components/gltf-model-plus.js

```diff
@@ -161,18 +165,30 @@ const inflateEntities = function(node, templates, isRoot, modelToWorldScale = 1)
     node.parent.animations = node.animations;
   }
 
-  if (entityComponents) {
-    for (const prop in entityComponents) {
-      if (entityComponents.hasOwnProperty(prop) && AFRAME.GLTFModelPlus.components.hasOwnProperty(prop)) {
-        const { componentName, inflator } = AFRAME.GLTFModelPlus.components[prop];
-        inflator(el, componentName, entityComponents[prop], entityComponents);
-      }
-    }
+  const gltfIndex = node.userData.gltfIndex;
+  if (gltfIndex !== undefined) {
+    indexToEntityMap[gltfIndex] = el;
   }
 
   return el;
 };
 
+function inflateComponents(inflatedEntity, indexToEntityMap) {
+  inflatedEntity.object3D.traverse(object3D => {
+    const entityComponents = getHubsComponents(object3D);
+    const el = object3D.el;
+
+    if (entityComponents && el) {
+      for (const prop in entityComponents) {
+        if (entityComponents.hasOwnProperty(prop) && AFRAME.GLTFModelPlus.components.hasOwnProperty(prop)) {
+          const { componentName, inflator } = AFRAME.GLTFModelPlus.components[prop];
+          inflator(el, componentName, entityComponents[prop], entityComponents, indexToEntityMap);
```

Comment:
Having this `indexToEntityMap` for inflators feels nice, that way all references inside the GLTF can now be index based, makes them feel much more "native" to the format.